assets/calendar/calendar.js

```diff
@@ -51,10 +62,11 @@ $(document).ready(function () {
             $("#description").text(calEvent.description);
```

Comment:
can we use calEvent without defining it?

Reply:
We cannot, but we can get the description if that’s your goal.